runtime/gateway.go

```diff
@@ -452,6 +452,9 @@ func (gateway *Gateway) setupMetrics(config *StaticConfig) (err error) {
 	defaultTags := map[string]string{
 		"env":     env,
 		"service": service,
+		"type":    metricsType,
```

Comment:
This tag is redundant. Usually only M3 metrics are emitted with tags. Would you ever query on `type` field?

Reply:
fixed